app/lib/http/index.js

```diff
@@ -0,0 +1,75 @@
+import { merge, pickBy } from 'lodash';
+import qs from 'qs';
+
+const JSON_HEADERS = {
+  'Accept': 'application/json',
+  'Content-Type': 'application/json'
+};
+
+const defaultPayload = () => ({
+  method: 'GET',
```

Comment:
Could we remove this attribute? If I'm not mistaken below you're defining `method` for each type of requests